src/express/commands/chaos.js

```diff
@@ -33,7 +33,7 @@ async function removeAllPeers(ip, staticNodes) {
 async function addAllPeers(ip, staticNodes) {
   const tasks = []
   for (let i = 0; i < staticNodes.length; i++) {
-    const command = `/home/ubuntu/go/bin/bor attach ~/.bor/data/bor.ipc --exec "admin.addPeer('${staticNodes[i]}')"`
+    const command = `/home/ubuntu/go/bin/bor attach /var/lib/bor/data/bor.ipc --exec "admin.addPeer('${staticNodes[i]}')"`
```

Comment:
And here as well.

Reply:
Done, [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/80a4181e29019ffe3b0d5454b56f13689cff4f33).